File: JUNGOL/Beginner/#2514 문자열 찾기/문자열 찾기.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	string s; cin>>s;

	int koi=0, ioi=0;
	for(int i=0; i<s.length()-2; i++){
		if(s.substr(i, 3) == "KOI") koi++;
		else if(s.substr(i, 3) == "IOI") ioi++;
	}

	cout<<koi<<endl;
	cout<<ioi<<endl;
}

File: JUNGOL/Beginner/#2514 문자열 찾기/문자열 찾기.py
def main():
    s = input()

    koi,ioi = 0,0
    for i in range(len(s)-2):
        if s[i:i+3] == "KOI": koi += 1
        elif s[i:i+3] == "IOI": ioi += 1

    print(koi, ioi, sep='\n')

main()
